.github/workflows/go.yml

```
@@ -306,11 +306,13 @@ jobs:
         run: brew install apache-arrow pkg-config
       - name: Install staticcheck
         run: go install honnef.co/go/tools/cmd/staticcheck@${{ matrix.staticcheck }}
-      - name: Build
+      - name: Add To pkg config path
         shell: bash
         run: |
-           echo "PKG_CONFIG_PATH=/usr/local/Cellar/openssl@3/3.0.8/lib/pkgconfig:$PKG_CONFIG_PATH" >> $GITHUB_ENV
-          ci/scripts/go_build.sh $(pwd)
+          echo "PKG_CONFIG_PATH=/usr/local/Cellar/openssl@3/3.0.8/lib/pkgconfig:$PKG_CONFIG_PATH" >> $GITHUB_ENV
```

How about using `brew --preifx` to get `openssl@3` path?

```suggestion
          echo "PKG_CONFIG_PATH=$(brew --prefix openssl@3)/lib/pkgconfig:$PKG_CONFIG_PATH" >> $GITHUB_ENV
```

---

i didn't know that was a thing! Thanks much @kou!